### PR TITLE
docs: CI setup

### PR DESCRIPTION
**Pull request by mythi**

### CODE REVIEW

**Comment by dcmiddle** [comment]
Nice initiative to document this so it doesn't become forgotten secret lore. :-D

File: docs/ci-setup.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+$ tail -3 /etc/containerd/config.toml
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.enclavecc]
+  cri_handler = "cc"
+  runtime_type = "io.containerd.rune.v2"
+```
+
+# verdictd (for EAA-KBC testing)
+- install [verdictd](https://github.com/inclavare-containers/verdictd) and its dependencies
+- [RATS-TLS](https://github.com/inclavare-containers/rats-tls) version is recommended to be the same as what the `agent-enclave` build uses.
+- `/usr/local/bin/verdictd --listen 127.0.0.1:1234 --verifier sgx_ecdsa --attester nullattester --client-api 127.0.0.1:12340 --mutual`
```
Comment (Xynnn007) [inline docs/ci-setup.md:25]: verdictd used for https://github.com/confidential-containers/enclave-cc/pull/140
```bash
docker run  \
    -v $(pwd)/data:/opt/verdictd \
    -v /etc/sgx_default_qcnl.conf:/etc/sgx_default_qcnl.conf \
    -v /usr/local/lib/rats-tls:/usr/local/lib/rats-tls \
    --device /dev/sgx_enclave \
    --device /dev/sgx_provision \
    -p 12345:12345 \
    --env RUST_LOG=debug \
    -d xynnn007/verdictd:cc0.5.0-rc0 \
    verdictd --listen 0.0.0.0:12345 --client-api 127.0.0.1:50000 --verifier sgx_ecdsa --attester nullattester --mutual 
```

which relys on the `rats-tls` on the host.

Comment (mythi) [reply docs/ci-setup.md:25]: OK..see the comment about `ACTIONS_RUNNER_HOOK_JOB_STARTED` below. I think that conflicts with this.

Comment (Xynnn007) [reply docs/ci-setup.md:25]: Oh yes. Actually there is a verdictd daemon always running on the CI machine, thus all tests set the `"key_provider"` to `"provider:attestation-agent:eaa_kbc::127.0.0.1:1234"`.

Now I started a new verdictd and listened to port `12345`.

The start-up and shut-down of the verdictd is not in the previous CI, neither the new CI

**Comment by Xynnn007** [comment]
Hi Mikko, I've edited an PR to add steps for verdictd and kbs here. Please take a look if it is good for you https://github.com/mythi/enclave-cc/pull/16

**Comment by mythi** [comment]
rebased and pushed with updates from @Xynnn007

File: docs/ci-setup.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+```bash
+mkdir -p $VERDICTD_WORKDIR/data
+docker run -d -v /etc/sgx_default_qcnl.conf:/etc/sgx_default_qcnl.conf \
+    -v $VERDICTD_WORKDIR/data:/opt/verdictd \
+    --device /dev/sgx_enclave \
+    --device /dev/sgx_provision \
+    -p 12345:12345 \
+    --env RUST_LOG=debug \
+    xynnn007/verdictd:v0.5.0-rc1 \
```
Comment (mythi) [inline docs/ci-setup.md:36]: @Xynnn007 did we make `Dockefile`s available for this and kbs/as below?

Comment (Xynnn007) [reply docs/ci-setup.md:36]: Yes. Verdictd (attention: are in CC branch)
- dockerfile https://github.com/inclavare-containers/verdictd/blob/CC/Dockerfile
- source code: https://github.com/inclavare-containers/verdictd/tree/CC

AS:
- Dockerfile & source code are in AS' repo. As occlum verifier was merged after v0.5.0, I had made a new image `docker.io/xynnn007/attestation-service:sgx-v0.6.0` based on the commit `32d0cbf5baf93566f7819981ec9e40e5c6c9ba55` https://github.com/confidential-containers/attestation-service/commit/32d0cbf5baf93566f7819981ec9e40e5c6c9ba55

Comment (mythi) [reply docs/ci-setup.md:36]: thanks! I added these links to the doc too.

**Review by Xynnn007 — APPROVED** [review]

**Comment by mythi** [comment]
dropped eaa-kbc instructions. let's merge

**Review by dcmiddle — APPROVED** [review]

**Pull request merged**